components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/CustomTimeRangeSelector.jsx

```diff
@@ -113,9 +112,8 @@ export default class CustomTimeRangeSelector extends React.Component {
 
     handleEndTimeChange(date) {
         this.endTime = date;
-        const start = Moment(this.startTime);
-        const end = Moment(this.endTime);
-        if(end.diff(start) < 0) {
+        if(Moment(this.startTime,'YYYY-MM-DD HH:mm:ss.000').unix() >=
```

Comment:
Format code

Reply:
handleStartTimeChange

Reply:
This code has been refactored with PR #270